### ISSUE TITLE
New rules: Forbid element selectors and :deep in css modules

### ISSUE THREAD

**Issue by ilyub** [vuejs/eslint-plugin-vue#1805]
**Please describe what the rule should do:**

Forbid element selectors and :deep in css modules

**What category should the rule belong to?**

[ ] Enforces code style (layout)
[X] Warns about a potential error (problem)
[ ] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**

```vue
<style lang="scss" module>
// Bad
div { ... }

// Bad
.someClass :deep(div) { ... }
</style>
```

The first will have global effect, which is likely not desired, because "module" is a replacement of "scoped".

The second will not work at all. It will work correctly without :deep.

**Additional context**

All is described above

**Comment by ota-meshi** [comment]
Thank you for posting this issue.
eslint-plugin-vue doesn't parse CSS, so we can't check it.
I think you need to look for another eslint plugin or [stylelint](https://stylelint.io/) plugin, or implement a new plugin.
